assets/keymaps/default-linux.json

```diff
@@ -32,7 +32,7 @@
     }
   },
   {
-    "context": "menu",
+    "context": "menu || Picker",
```

Comment:
On an unrelated note, it seems like `menu` is the only key context that's in lowercase. Should this be changed?